components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -255,7 +255,7 @@ bool ResetOnFileTaskRunner(const base::FilePath& path) {
 bool ResetOnFilesTaskRunner(const std::vector<base::FilePath>& paths) {
   bool res = true;
   for (size_t i = 0; i < paths.size(); i++) {
-    if (!base::DeleteFile(paths[i], false)) {
+    if (!base::DeleteFile(paths[i], true)) {
```

Comment:
Just set flag to remove all files recursively

Reply:
Note from `base/files/file_util.h`

```cpp
// Note: The |recursive| parameter is in the process of being removed. Use
// DeleteFileRecursively() instead. See https://crbug.com/1009837
BASE_EXPORT bool DeleteFile(const FilePath& path, bool recursive);
```

Reply:
thanks @zenparsing, will change it